src/types.rs

```diff
@@ -245,10 +245,6 @@ pub(crate) mod serde_opt_date_from_unix_timestamp {
             .map(|timestamp| DateTime::from_utc(NaiveDateTime::from_timestamp(timestamp, 0), Utc)))
     }
 
-    pub(crate) fn none<T>() -> Option<T> {
-        None
-    }
```

Comment:
`none` is still used in the test down below

Reply:
Fixed